packages/editor-ui/src/components/ParameterInputList.vue

```diff
@@ -460,7 +458,7 @@ export default defineComponent({
 			const nodeValues: INodeParameters = {};
 			let rawValues = this.nodeValues;
 			if (this.path) {
-				rawValues = get(this.nodeValues, this.path);
+				rawValues = get(this.nodeValues, this.path) as INodeParameters;
```

Comment:
Yes, but it's hard because the original type is

```
type NodeParameterValueType = NodeParameterValue | INodeParameters | INodeParameterResourceLocator | ResourceMapperValue | FilterValue | AssignmentCollectionValue | NodeParameterValue[] | INodeParameters[] | INodeParameterResourceLocator[] | ResourceMapperValue[]
```

it's hard to write a type guard for `INodeParameters` because it had no unique properties, and a lot of other types are also objects.